02-express-tutorial/routes/auth.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const router = express.Router();
+
+
+router.post('/', (req, res) => {
+  const { name } = req.body;
+  if (name) {
+    return res.status(200).send(`Welcome, ${name}!`);
```

Comment:
Challenge: can we capitalize peoples' names here, even if the name we receive has a lowercase first letter?

Reply:
Yes, we can manipulate the received data. 
To make the first letter of the name capitalized we can do next: 
name = name[0].toUpperCase().concat(name.slice(1))

Reply:
correction: ```name.split(' ')
  .map(letter => letter[0].toUpperCase() + letter.slice(1))
  .join(' ')```